+Prefs/+Inputs/LabelControlBasic.m

```diff
@@ -18,10 +18,16 @@
         % Important to do it this way to allow MATLAB to make the proper extent
         %   when generating the label uicontrol for "label_width_px"
         function labeltext = get_label(~,pref)
+            name = pref.name;
+            
+            if isempty(name)
+                name = strrep(pref.property_name, '_', ' ');
+            end
+            
             if ~isempty(pref.units)
-                labeltext = sprintf('%s (%s)',pref.name,pref.units);
+                labeltext = sprintf('%s (%s)', name, pref.units);
```

Comment:
This should be changed to square brackets.

Reply:
lol sure